tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-on-device-training.yml

```diff
@@ -0,0 +1,48 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_GPU_OnDeviceTraining
```

Comment:
do you think we can put CPU ep tests on this pipeline or we should have a seperate CPU pipeline?

Reply:
I think we can put the cpu tests in this pipeline itself. We don't need to build ort two times and consume resources.